data/bids_layout/participants.json

```diff
@@ -0,0 +1,18 @@
+{
+    "participant_id": {
+        "Description": "Unique participant identifier"
+    },
+    "age": {
+        "Description": "Age of the participant at time of testing",
+        "Units": "years"
+    },
+    "sex": {
+        "Description": "Biological sex of the participant",
+        "Levels": {
+            "F": "female",
+            "M": "male"
+        }
+    },
+    "resected_contacts": "The list of contacts corresponding to resected regions",
```

Comment:
These don't follow the guidelines that the other fields have:

E.g. "Description", and then "Units/Levels/etc." If there is nothing after Description, we just don't add it.